packages/common/contact.ts

```diff
@@ -98,6 +98,32 @@ const toAllowableMethod = (contact: ContactData): PartialContactMethod => {
   }
 }
 
+/**
+ * It's possible to decide whether to increment old emails or replace them
+ * entirely
+ */
+interface ContactDataOverride extends Partial<ContactData> {
+  /** Defaults to replace when undefined */
+  emailOverrideMethod?: 'increment' | 'replace'
```

Comment:
Nit: let's add one for `faxes` as well.  Let's make them `'join' | 'replace'`.

But let's just remove this -- we don't need it since we do not yet `join` any data.